packages/graphql-validate-fixtures/src/index.ts

```diff
@@ -1,68 +1,168 @@
 import {readFile, readJSON} from 'fs-extra';
-import {isAbsolute, resolve} from 'path';
+import {resolve} from 'path';
+import * as glob from 'glob';
+import {Source, parse, concatAST, GraphQLSchema} from 'graphql';
+import {getGraphQLConfig, GraphQLProjectConfig} from 'graphql-config';
+import {AST, compile, Operation} from 'graphql-tool-utilities/ast';
 import {
-  GraphQLSchema,
-  buildClientSchema,
-  Source,
-  parse,
-  concatAST,
-} from 'graphql';
-import {compile} from 'graphql-tool-utilities/ast';
+  getGraphQLFilePath,
+  getGraphQLProjects,
+} from 'graphql-tool-utilities/config';
 
 import {
+  getOperation,
   validateFixtureAgainstAST,
-  validateFixtureAgainstSchema,
   Validation,
   Fixture,
+  MissingOperationError,
 } from './validate';
 
-export interface Paths {
-  fixturePaths: string[];
-  operationPaths?: string[];
-  schemaPath: string;
-}
-
 export interface Options {
-  schemaOnly?: boolean;
+  cwd: string;
 }
 
 export interface Evaluation extends Validation {
-  fixturePath: string;
   scriptError?: Error;
 }
 
 export async function evaluateFixtures(
-  {fixturePaths, operationPaths = [], schemaPath}: Paths,
-  {schemaOnly = false}: Options = {},
+  fixturePaths: string[],
+  {cwd}: Options,
 ): Promise<Evaluation[]> {
-  let schema: GraphQLSchema;
+  const config = getGraphQLConfig(cwd ? resolve(cwd) : undefined);
```

Comment:
Doesn't look like this ternary is needed, cwd will always be truthy.

Reply:
good catch! It was previously not the case, but now i can remove this check.